### PR TITLE
[accounts#find] documentation

### PR DESCRIPTION
**Pull request by macejmic**
**What this PR does / why we need it**:

Documentation for existing functionality. 

**Which issue(s) this PR fixes** 

https://issues.jboss.org/browse/THREESCALE-1562

### CODE REVIEW

File: app/controllers/admin/api/accounts_controller.rb [hunk 1]
```diff
@@ -258,9 +262,9 @@ def find_buyer_account
       buyer_users.find_by!(username: username).account
     when user_id = params[:user_id]
       buyer_users.find(user_id).account
-    when current_account.master? && provider_key = params[:buyer_provider_key]
```
Comment (hallelujah) [inline app/controllers/admin/api/accounts_controller.rb]: There is no such developer provider key ... So not really needed to remove this check.
The idea behind is if I understand for documentation only?

Either we duplicate this logic/controller into https://github.com/3scale/porta/tree/121a66ca7566cbb33e5ed9460efb9ae007f43cba/app/controllers/master/api

Duplicating would be a bad idea as many APIs are shared between provider and master
Another solution is to have a different json file for documentation served depending if you are master or provider but that is overkill IMHO. https://github.com/3scale/porta/blob/121a66ca7566cbb33e5ed9460efb9ae007f43cba/app/controllers/api_docs/services_controller.rb#L59-L65

File: app/controllers/admin/api/accounts_controller.rb [hunk 2]
```diff
@@ -57,11 +57,15 @@ def index
   ##~ @parameter_username = {:name => "username", :description => "Username of the account user.", :dataType => "string", :paramType => "query"}
   ##~ @parameter_email = {:name => "email", :description => "Email of the account user.", :dataType => "string", :paramType => "query"}
   ##~ @parameter_id = {:name => "user_id", :description => "ID of the account user.", :dataType => "integer", :paramType => "query"}
+  ##~ @parameter_provider_key = {:name => "buyer_provider_key", :description => "[Master API] Provider key of the account.", :dataType => "string", :paramType => "query"}
```
Comment (josemigallas) [inline app/controllers/admin/api/accounts_controller.rb]: Why not `@parameter_buyer_provider_key`?

Comment (macejmic) [reply app/controllers/admin/api/accounts_controller.rb]: well, it does not matter that much.. it's just a "variable" name.

Comment (josemigallas) [reply app/controllers/admin/api/accounts_controller.rb]: I'd say be consistent, just in case.

**Pull request merged**